### PR TITLE
Add an audio dependency(Asteroid)

### PR DESCRIPTION
**Pull request by hihunjin**
## What does this PR do?

Add an audio dependency(Asteroid)

Fixes # (issue)

## Before submitting
- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/lightning-flash/tree/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? [not needed for typos/docs]
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/lightning-flash/blob/master/CHANGELOG.md)?

## PR review
 - [x] Is this pull request ready for review? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

We've discussed this pr on our slack channel.

## Did you have fun?
Make sure you had fun coding 😄

### CODE REVIEW

**Review by Borda — CHANGES_REQUESTED** [review]
we cannot add this dependency as it creates a cycle: Asteroid > PL > TM
moreover, the requested version is very resting PT versions
https://github.com/asteroid-team/asteroid/blob/ee1ff2447b5cceb6d0df006cb33e7fee9235ea23/requirements/torchhub.txt#L5

**Comment by hihunjin** [comment]
> ignore me here... @hihunjin

What was TM and PT? PT=PyTorch version.
I guess you did a good mention. what if like
```sh
asteroid-filterbanks>=0.4.0
requests
filelock
SoundFile>=0.10.2
huggingface_hub>=0.0.2
```
on requirement?

**Comment by Borda** [comment]
> I guess you did a good mention. what if like

no, it is fine, FLash requires PyTorch 1.8 as well as Asteroid
https://github.com/PyTorchLightning/lightning-flash/blob/f7a86eae45786db8ef56b8b24f3e28e13d2be581/requirements.txt#L1

File: flash/core/utilities/imports.py [hunk 1]
```diff
@@ -102,13 +104,18 @@ def _compare_version(package: str, op, version) -> bool:
     _PYSTICHE_AVAILABLE,
 ])
 _SERVE_AVAILABLE = _FASTAPI_AVAILABLE and _PYDANTIC_AVAILABLE and _CYTOOLZ_AVAILABLE and _UVICORN_AVAILABLE
+_AUDIO_AVAILABLE = all([
```
Comment (tchaton) [inline flash/core/utilities/imports.py:107]: Neat ! Good start :)

**Review by ethanwharris — APPROVED** [review]
Awesome, LGTM :smiley:

**Review by tchaton — APPROVED** [review]
LGTM !

**Review by akihironitta — APPROVED** [review]

**Pull request merged**